### ISSUE TITLE
How do you calculate the val_acc & val_loss? Is it from test dataset or from the training dataset?

### ISSUE THREAD

**Issue by albararamli** [OlafenwaMoses/ImageAI#388]
How do you calculate the validation? Is it from the test dataset or from the training dataset?

**Comment by OlafenwaMoses** [comment]
Validation accuracy is calculated from:
- images from your `test` folder in custom recognition training
- images from your `validation` folder in custom detection training
